### DISCUSSION TITLE
How to keymap j to ArrowDown key ?

### DISCUSSION THREAD

**Discussion by Akianonymus** [tridactyl/tridactyl#4327] [Q&A]
Well scrolling done by tridactyl is very janky, so wanted to map my j and k key to up and down arrow.

**Comment by bovine3dom** [comment]
You can only do this with an external programme like xdotool and the native messenger (`:native`). Then it should be as simple as `:bind j !s xdotool key Down`

**Comment by Akianonymus** [comment]
First, that doesn't work on holding the key.

Second, that will just xdotool everytime.

**Reply by bovine3dom** [reply]
Yeah, for holding down the key we'd need to merge #1645.

I don't understand what you mean by your second point, could you rephrase it?

**Reply by Akianonymus** [reply]
Everytime j is pressed, xdotool program will be called.

**Reply by bovine3dom** [reply]
Yes, that's the point. You press j, Tridactyl calls xdotool, xdotool fires a down arrow key event.

Firefox does not allow us to fire real key events so we must use an external tool.
